integration-test/utils/datafactory/schemas/pagesection.schema.ts

```diff
@@ -0,0 +1,32 @@
+import { linkSchema } from './link.schema';
+
+export const pageSectionSchema = {
+  $ref: '#/definitions/pageSectionSchema',
+  definitions: {
+    pageSectionSchema: {
+      type: 'object',
+      properties: {
+        title: {
+          type: 'string',
```

Comment:
Please ensure that there could be no empty string in the response, by providing additional rule in schema. Also, please, test different scenarios to check if schema covers all of them.